expression/builtin_string.go

```diff
@@ -2136,55 +2136,82 @@ type makeSetFunctionClass struct {
 	baseFunctionClass
 }
 
+func (c *makeSetFunctionClass) getFlen(sc *variable.StatementContext, args []Expression) int {
+	flen, count := 0, 0
+	if constant, ok := args[0].(*Constant); ok {
+		bits, isNull, err := constant.EvalInt(nil, sc)
+		if err == nil && !isNull {
+			for i, length := 1, len(args); i < length; i++ {
```

Comment:
It looks like `length` will be changed in the for loop, but actually not. The same with others in this pr. May be looks good for every one else, just comment here.